src/Pool.js

```diff
@@ -56,6 +75,7 @@ function Pool(script, options) {
     this._ensureMinWorkers();
   }
 
+  /** @private */
   this._boundNext = this._next.bind(this);
```

Comment:
I think the `@private` and `@protected` annotations on methods are bit mixed (that already was the case). Shall we make all internal methods in `Pool.js` `@private` (they are prefixed with a `_`)?